src/components/authoring.tsx

```diff
@@ -168,14 +183,21 @@ export const defaultAuthoring = {
         tan: 33.33
       }
     }
+  },
+  organism: {
```

Comment:
@sfentress We're going to need to work with Michael to determine how to show the Organism level instructions.  For now, they exist in the authoring tool, but they will not be placed anywhere until we settle on how the instructions would be displayed in the 4-up.